_includes/javascripts.html

```diff
@@ -1,9 +1,13 @@
 {% unless site.local_fonts %}
-  <script src="https://ajax.googleapis.com/ajax/libs/webfont/1.6.16/webfont.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/webfont/1.6.27/webfontloader.js"></script>
```

Comment:
I show you all the change I did and yes that get rid of the error. But have 1 weird thing too but this is happening before too it's when I see a blog post some time if I go back I just see 1 in my post list all the other don't came. I really thing the ScrollReveal it's the problem